### ISSUE TITLE
rebasing：when rebase client to master，where is C3?

### ISSUE THREAD

**Issue by tiankonguse** [progit/progit2#1117]
https://git-scm.com/book/en/v2/Git-Branching-Rebasing#_more_interesting_rebases

in here, we rebase client to master, the book say that c8 & c9 should recommit to master.
if this, master don't have c3 commit's function。
it should 

![https://git-scm.com/book/en/v2/images/interesting-rebase-1.png](https://git-scm.com/book/en/v2/images/interesting-rebase-1.png)
![https://git-scm.com/book/en/v2/images/interesting-rebase-2.png](https://git-scm.com/book/en/v2/images/interesting-rebase-2.png)

**Comment by ben** [comment]
You're right! At some point I'll convert all the figures from Sketch to SVG, and then these kinds of issues will be much easier to fix.

**Comment by YueLinHo** [comment]
> I'll convert all the figures from Sketch to SVG

@ben Are you working on it?

**Comment by Quivelus** [comment]
Found this issue when wanting to share the same thought, that is: were is C3 --> C3'? Maybe one addition to this is that it might be more clear to add a double quote to annotate the Server rebase to distinguish between the client rebase. For example by naming the Server commits C3", C4" and C10" respectively.
